packages/grafana-ui/src/components/TimePicker/TimePickerContent/TimePickerContent.tsx

```diff
@@ -263,7 +263,11 @@ const EmptyRecentList = memo(() => {
         </span>
       </div>
       <div>
-        <a className={styles.link} href="https://grafana.com/docs/grafana/latest/reference/timerange/" target="_new">
+        <a
+          className={styles.link}
+          href="https://grafana.com/docs/grafana/latest/dashboards/time-range-controls"
```

Comment:
Something wrong with docs aliases, highlighted in #docs channel, the old url should till work. Seems a lot of docs aliases are not working

Reply:
Ah ok.